app/resources/v1/pacbio/run_resource.rb

```diff
@@ -63,7 +63,7 @@ def publish_messages
 
       private
 
-      def plates_attributes=(plates_parameters)
+      def plates_attributes=(plates_parameters) # rubocop:disable Metrics/MethodLength
```

Comment:
Is this method length disable still needed?

Reply:
It seems so, Rubocop is complaining about "Metrics/MethodLength: Method has too many lines."